ex4q3output.txt

```diff
@@ -0,0 +1,3 @@
+yearsSchool wage
+16 9.4361877801
+12 0.1028907398
```

Comment:
We're looking for the difference here, but good job getting the important values.